docs/accounts/Opening-Requirements.md

```diff
@@ -1,4 +1,5 @@
 ---
+stoplight-id: cb46e29ea99a4
```

Comment:
Added this (and used the existing, generated ID from our live stoplight guides) to allow me to link to the corporate GIA section.

Reply:
See [this page](https://docs.stoplight.io/docs/platform/hosted-docs/stoplight-urls#about-relative-links) for more info.